ui/app/components/auth-form.js

```diff
@@ -25,9 +27,21 @@ export default Ember.Component.extend({
     return `auth-form/${type}`;
   }),
 
+  hasCSPError: computed.alias('csp.connectionViolations.firstObject'),
+
+  cspErrorText: `This is a standby Vault node but can't communicate with the active node via request forwarding. Sign in at the active node to use the Vault UI.`,
```

Comment:
Nit: there's no templated code in here or in the string used in the tests below. Maybe you're just using it since the `can't` is in there?

Reply:
Yeah originally it was multi-line because it's a beefy error, but then I took that out so I didn't have to trim whitespace in tests 😂. THEN, when I went to single quote I noticed the `can't` and decided to leave the `s.